core/src/main/java/io/fabric8/maven/core/service/kubernetes/DockerBuildService.java

```diff
@@ -57,6 +61,11 @@ public void build(ImageConfiguration imageConfig) throws Fabric8ServiceException
         }
     }
 
+    @Override
+    public void build(JibBuildConfigurationUtil buildConfigurationUtil) throws Fabric8ServiceException, InvalidImageReferenceException, InterruptedException, RegistryException, IOException,  ExecutionException {
```

Comment:
Removes all exceptions and in the body of the method just throw `new UnsupportedOperationExcetion()`